systemtest/fleet_test.go

```diff
@@ -153,6 +153,26 @@ func TestFleetIntegrationAnonymousAuth(t *testing.T) {
 	test("denied_service", "allowed_agent", http.StatusForbidden)
 }
 
+func TestFleetIntegrationSourcemap(t *testing.T) {
```

Comment:
Adding a systemtest to make sure sourcemapping is working with fleet. Does it make sense to have this ?

Reply:
I think this is a good idea for now, but maybe later we can make it a smoke test.